src/sas/qtgui/Utilities/ResultPanel.py

```diff
@@ -94,6 +97,12 @@ def onPlotResults(self, results, optimizer="Unknown"):
         if self.count()==0:
             self.close()
 
+    def dataDeleted(self, data):
```

Comment:
It would probably be better to call this method `onDataDeleted` to signify it is a response to the action.